### PR TITLE
📦 Next

### PR DESCRIPTION
**Pull request by stoe**

### CODE REVIEW

**Review by digimangos — COMMENTED** [review]
Two things...

  - shouldn't this inherit `debug` from `@actions/code` and use `debug(error.stack)` for output?
  - I tried this whilst I was debugging, but of course when running as an action it's using the compiled/minified distribution, which then isn't very helpful.

File: action.js [hunk 1]
```diff
@@ -47,6 +47,11 @@ import ActionPolicy from './utils/ActionPolicy.mjs'
     info(`✅ Updated GitHub Actions allow list for ${enterprise || organization}`)
     setOutput('output', `GitHub Actions allow list updated for ${enterprise || organization}`)
   } catch (error) {
+    if (isDebug()) {
+      // print stack trace
+      console.error(error.stack)
+    }
+
```
Comment (stoe) [inline action.js:54]: @digimangos, you referring to this in https://github.com/ActionsDesk/github-actions-allow-list-as-code-action/pull/82#pullrequestreview-1483152335?

I found `debug()` (https://github.com/actions/toolkit/blob/a6bf8726aa7b78d4fc8111359cca5d538527b239/packages/core/src/core.ts#L237-L243) to not ever returning anything, so looking to rely on core `console` usage wrapped into [`isDebug()`](https://github.com/actions/toolkit/tree/main/packages/core#logging) check here.

Comment (stoe) [reply action.js:54]: Additional context `debug()` only allows `String` as input, thus also not usable to print the stack trace.

**Review by mouismail — APPROVED** [review]
LGTM away from CodeQL alert

**Comment by stoe** [comment]
> LGTM away from CodeQL alert

That ones is also addressed, as it was only related to the generated code in `dist/`

**Pull request merged**